Assets/BossRoom/Scripts/Shared/Net/ConnectionManagement/ClientGameNetPortal.cs

```diff
@@ -238,7 +238,7 @@ public async void StartClientUnityRelayModeAsync(GameNetPortal portal, string jo
                         var clientRelayUtilityTask = UnityRelayUtilities.JoinRelayServerFromJoinCode(joinCode);
                         await clientRelayUtilityTask;
                         var (ipv4Address, port, allocationIdBytes, connectionData, hostConnectionData, key) = clientRelayUtilityTask.Result;
-                        utp.SetRelayServerData(ipv4Address, port, allocationIdBytes, key, connectionData, hostConnectionData);
+                        utp.SetClientRelayData(ipv4Address, port, allocationIdBytes, key, connectionData, hostConnectionData, true);
```

Comment:
for clarity in a sample like this, I usually like to have named parameters for simple types, so readers know what that "true" is useful for
```suggestion
                        utp.SetClientRelayData(ipv4Address, port, allocationIdBytes, key, connectionData, hostConnectionData, isSecure: true);
```

Reply:
For the same reason [mentioned here](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop/pull/485#discussion_r812967013), I think UTP won't really have a choice but to adopt the string-based (or enum) connection types instead of relying on a boolean.

Reply:
Good point, although unfortunately I'm not sure we can/want to break the API at this point. There's also no plan to support TCP in UTP (except through WebSockets eventually), so for now I think we can live with the limitation.